### ISSUE TITLE
error in layouts

### ISSUE THREAD

**Issue by alancivale** [Aymkdn/html-to-pdfmake#101]
with this change in the file: index.js line 291 we cover the conversion error due
to incorrect characters. avoiding you have to put:
```html
<table data-pdfmake = "{& quot; layout & quot ;: & quot; exampleLayout & quot;}">
```

for something simpler:
```html
<table data-pdfmake='{"layout":"noBorders"}'> 
<table data-pdfmake="{'layout':'noBorders'}"> 
```
```js
// check if we have some data-pdfmake to apply
            if (element.dataset && element.dataset.pdfmake) {
              try{
                dataset = JSON.parse(element.dataset.pdfmake);
              }catch(e){
                if(element.dataset.pdfmake){
                  try{
                    element.dataset.pdfmake = element.dataset.pdfmake.replace(/'/g,'"');
                    dataset = JSON.parse(element.dataset.pdfmake);
                  }catch(e){
                    console.log(e);
                    dataset = {};
                  }
                }
              }
              for (key in dataset) {
                if (key === "layout") {
                  ret.layout = dataset[key];
                } else {
                  ret.table[key] = dataset[key];
                }
              }
            }
```

**Comment by alancivale** [comment]
data-pdfmake = "{& quot; layout & quot ;: & quot; exampleLayout & quot;}"

for something simpler:  
data-pdfmake='{"layout":"noBorders"}'
or
data-pdfmake="{'layout':'noBorders'}"

**Comment by Aymkdn** [comment]
I think I see what you mean. I'll think about it and check by the end of the week-end.

**Comment by alancivale** [comment]
If you are interested, I made some changes to make the html step easier, I made everything asynchronous, to be able to request the conversion and at the end it returns it to me, with this I made it possible to add images with a url, and I take care of download it, resize it and convert it to base64 automatically. and I added a cache in memory to reuse images if they request the same url and thus speed up if I send several pdfs with the same image.
it's great what you put together. sorry if english is wrong,
use the translator.
regards!
